app/components/watch-show.js

```diff
@@ -51,7 +57,7 @@ export default Component.extend(
       track({
         action: trackActions.impression,
         category: 'article',
-        label: 'watch-show',
+        label: `watch-${this.wikiVariables.watchShowTrackingLabel}`,
```

Comment:
Will it matter that we're not checking if `this.wikiVariables.watchShowTrackingLabel` is defined?

Reply:
71f799a